lib/puppet/application/cert.rb

```diff
@@ -132,7 +148,10 @@ def help
   List outstanding certificate requests. If '--all' is specified, signed
   certificates are also listed, prefixed by '+', and revoked or invalid
   certificates are prefixed by '-' (the verification outcome is printed
-  in parenthesis).
+  in parenthesis). If '--human-readable' or '-H' is specified listed
+  certificates are formatted in a way to improve human scan-ability. If
+  '--machine-readable' or '-m' is specified output is formatted in a
+  line-wise format optimal for scripted consumption.
```

Comment:
`line-wise` is a strange word to me, instead I suggest something like `... a concise format suitable for consumption by a script` or something like that.